injector/network_disruption_test.go

```diff
@@ -412,6 +413,21 @@ var _ = Describe("Failure", func() {
 				tc.AssertCalled(GinkgoT(), "AddFilter", []string{"lo", "eth0", "eth1"}, "1:0", mock.Anything, "nil", "8.8.8.8/32", 0, 53, "tcp", "", "1:1")
 			})
 		})
+
+		Context("with a re-injection", func() {
+			JustBeforeEach(func() {
+				// When an update event is sent to the injector, the disruption method Clean is called before its Inject method.
+				// If the method Clean is not called the AddNetem operations will stack up.
+				Expect(inj.Clean()).To(BeNil())
+				Expect(inj.Inject()).To(BeNil())
```

Comment:
So this aims to mimic a re-injection behaviour?

Reply:
Exactly 👍